scrapy/extensions/feedexport.py

```diff
@@ -310,6 +312,8 @@ def finish_exporting(self):
 
 
 class FeedExporter:
+    pending_deferreds = []
```

Comment:
Any reason *not* to make it private?

```suggestion
    _pending_deferreds = []
```

We usually make things private unless we have a reason to make them public. It makes it easier to implement future changes by not being bound by backward compatibility promises.